### PR TITLE
Fixed tests and debug string

### PR DESCRIPTION
**Pull request by rashmihunt**
- Fixed tests to accommodate Fault response when external Weather service is down.
- Fixed debug string.
@raymondfeng PTAL

### CODE REVIEW

File: lib/http.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
 var req = require('request');
 var util = require('util');
 var url = require('url');
-var debug = require('debug')('node-soap');
+var debug = require('debug')('loopback:connector:http');
```
Comment (raymondfeng) [inline lib/http.js]: The debug string should be `loopback:connector:soap` or `loopback:connector:soap:http`.

Comment (rashmihunt) [inline lib/http.js]: Fixed

**Comment by rashmihunt** [comment]
@slnode test please

**Comment by rashmihunt** [comment]
@slnode test please

**Comment by rashmihunt** [comment]
@slnode test please

**Pull request merged**